### ISSUE TITLE
User Consent

### ISSUE THREAD

**Issue by ifi223665** [Ylianst/MeshCentral#4968]
Hello, 

First of all, I'd like to thank the team for their contribution on this very useful tool. 
I'm using meshcentral v1.1.0 and Everything is working quite well with the exception of the "User Consent".

The computers are in a domaine in the same OU; thus, inheriting the same GPOs.  

For some computers, the user consent warning pops accordingly and once the end user clicks on permit, I am able to connect to the computer. 
For some other computers, clicking permit or refused has absolutely no effect. Wireshark traces shows that no packet is sent after the user clicks "permit" or "refuse". Windows Firewall is not blocking either.

It is as if the action of clicking "permit" of "refuse" is not being translated properly to the system so that it can send the packet to the server. 

Thank you for your help,

John

**Comment by OutbackMatt** [comment]
Are they in the same 'device group' in MeshCentral Server?

**Comment by ifi223665** [comment]
yes indeed, they belong to the same device group where user consent is activated. Anyone knows of any type of low level debug that can be activated on the client side ?
thx

**Comment by dinger1986** [comment]
Is this still a problem?

**Comment by ifi223665** [comment]
Hello, not this is good now.
thank you
